### PR TITLE
Book detail

### PR DESCRIPTION
**Pull request by andyOlarte514**
## Summary ##

**trello cards**
- https://trello.com/c/J7lsAQV4
- https://trello.com/c/S6LeI2WZ
- https://trello.com/c/zxhGCt20

- View Book detail
- Rent book
- Book comments

### iPhone 5s

- 312de877b969849b9a57813643c928f6

### iPhone X/XS/XR

- 5ba1e70cd7f0a4462d61dc2e39811c64

### CODE REVIEW

File: WBooks/BookRepository.swift [hunk 1]
```diff
@@ -38,5 +43,46 @@ public class BookRepository {
             }
         }
     }
-    
+
+    public func fetchComments(book: Book, onSuccess: @escaping ([Comment]) -> Void, onError: @escaping (Error) -> Void) {
+        let url = URL(string: Constants.endpointComment(with: book.id))!
```
Comment (gmazzei) [inline WBooks/BookRepository.swift]: Use safe unwrapping here. For example `guard let url = URL(string: Constants.endpointComment(with: book.id)) else { return }`

File: WBooks/Screens/BookDetail/BookDetailView.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        }
+    }
+    @IBOutlet weak var bookDetailTableComments: UITableView!
+
+    func initBookDetailView(with bookDetail: Book) {
+        backgroundColor = .backgroundPolar()
+        bookDetailContainer.layer.cornerRadius = 5
+        if let url = URL(string: bookDetail.image) {
+            let resource = ImageResource(downloadURL: url)
+            bookDetailImg.kf.indicatorType = .activity
```
Comment (gmazzei) [inline WBooks/Screens/BookDetail/BookDetailView.swift]: What is `kf`?

File: WBooks/Extensions/UIDate.swift [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+//  WBooks
+//
+//  Created by Andy Francisco Olarte Cardona on 22/11/19.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import Foundation
+import WolmoCore
+
+extension Date {
```
Comment (gmazzei) [inline WBooks/Extensions/UIDate.swift]: Perhaps you could call the function `getString()` and remove the `static` keyword. For example:
```Swift
extension Date {
    func getString() -> String {
        let dateFormatter = DateFormatter()
        dateFormatter.locale = Locale.current
        dateFormatter.dateFormat = "yyyy-MM-dd"
        return dateFormatter.string(from: self)
    }
}
```

File: WBooks/Screens/BookDetail/BookDetailView.swift (hunk 2 above)
Comment (andyOlarte514) [reply WBooks/Screens/BookDetail/BookDetailView.swift]: `kf` is `KingFisher` => https://github.com/onevcat/Kingfisher

**Pull request merged**